docs/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -112,8 +126,29 @@ This flag is only supported from the V2 version of the provider options struct w
 
 Default value: 0
 
+### gpu_external_alloc
+### gpu_external_free
+### gpu_external_empty_cache
+### tunable_op_enable
+### tunable_op_tuning_enable
+### tunable_op_max_tuning_duration_ms
```

Comment:
@hariharans29 Who can help write some documentation for these options ?

Reply:
gpu_external_* is to pass address of external allocator APIs for allocating memory, free memory, can empty cache.

AFAIK, tunable ops options are placeholder since there are no tunable ops in CUDA EP.